jina/resources/completions/jina.bash

```diff
@@ -21,5 +21,7 @@ complete -F _jina jina
 # session-wise fix
 ulimit -n 4096
 export OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES
+# default workspace for Executors
+export JINA_DEFAULT_WORKSPACE_BASE="${HOME}/.jina/executor-workspace"
 
```

Comment:
If a user changes this in their `.bashrc`, then the next install will overwrite that back to this value. Is that a problem?

Reply:
I'm now explaining this in the docs, should be ok